Source/Diagnostics/WarpXIO.cpp

```diff
@@ -525,8 +525,13 @@ WarpX::WritePlotFile () const
 #ifdef WARPX_USE_OPENPMD
     if (dump_openpmd){
         // Write openPMD format: only for level 0
-        std::string filename = amrex::Concatenate("diags/hdf5/data", istep[0]);
```

Comment:
@RemiLehe if I change this path, for consistency, to `diags/h5/data....h5` would that break many workflows?
I added a work-around to keep it the old way, but am afraid this might be a bit cumbersome to access in the long run. Just wondering.

Reply:
I think it's fine to break this, and replace it by `diags/h5`. 
As far as I now, we do not have any tools that explicitly rely on this being `diags/hdf5`.

Reply:
excellent, then I will unify this for simplicity